apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/CharSequenceContainsRegexAssertionsSpec.kt

```diff
@@ -6,31 +6,54 @@ import org.spekframework.spek2.Spek
 
 class CharSequenceContainsRegexAssertionsSpec : Spek({
     include(StringSpec)
+    include(RegexSpec)
 }) {
     object StringSpec : ch.tutteli.atrium.specs.integration.CharSequenceContainsRegexAssertionsSpec(
         getNameContainsRegex(),
-        getAtLeastTriple(),
-        getAtLeastIgnoringCaseTriple(),
-        getShortcutTriple(),
-        getAtMostTriple(),
-        getAtMostIgnoringCaseTriple(),
+        getAtLeastTripleString(),
+        getAtLeastIgnoringCaseTripleString(),
+        getShortcutTripleString(),
+        getAtMostTripleString(),
+        getAtMostIgnoringCaseTripleString(),
+        "◆ ", "⚬ "
+    )
+
+    object RegexSpec : ch.tutteli.atrium.specs.integration.CharSequenceContainsRegexAssertionsSpec(
+        getNameContainsRegex(),
+        getAtLeastTripleRegex(),
+        getAtLeastIgnoringCaseTripleRegex(),
```

Comment:
use 
```suggestion
        getAtLeastIgnoringCaseTripleString(),
```

Reply:
I see you have problems spinning your head around this. Please just apply my suggestion, you don't need an own getAtLeastIgnoringCaseTripleRegex it is the same as getAtLeastIgnoringCaseTripleString()